src/patternfly/components/Sequence/sequence-content.hbs

```diff
@@ -0,0 +1,6 @@
+<span class="pf-c-sequence__content{{#if sequence-content--modifier}} {{sequence-content--modifier}}{{/if}}"
```

Comment:
nitpicky but wondering why this is a span? It seems more like a div to me if it's grouping things.

Reply:
Changed to a span so that the screen reader would read the content in order rather than breaking it up into chunks from the divs

Reply:
ah ok!